companies/serializers.py

```
@@ -1,14 +1,151 @@
+                for code_length in range(len(obj.induty_code) - 1, 0, -1):
+                    parent_code = obj.induty_code[:code_length]
+                    industry = Industry.objects.filter(
+                        induty_code=parent_code, is_deleted=False
+                    ).first()
+                    if industry:
+                        break
+
+            if industry:
+                return IndustrySerializer(industry).data
+        except Exception:
+            pass
```

Ruff (0.14.11)

`try`-`except`-`pass` detected, consider logging the exception

(S110)